domain_access/domain_access.module

```diff
@@ -30,6 +30,9 @@ define('DOMAIN_ACCESS_USER_FIELD', 'field_domain_user');
 function domain_access_node_grants(AccountInterface $account, $op) {
   $grants = array();
   $active = domain_get_domain();
+  
+  if(empty($active)){ $active = domain_default(); }
```

Comment:
If you use node_access_rebuild it will fail. It solves all the scenario's where domains are already set and a node_access_rebuild call is needed.